cmd/az-analyze/cmd/root.go

```diff
@@ -69,7 +67,7 @@ func initConfig() {
 	} else {
 		// Search config in home directory with name ".az-analyze" (without extension).
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("./config")
+		viper.AddConfigPath("/var/tmp/azuredisk-csi-driver/cmd/az-analyze/config")
```

Comment:
We should use the relative path, like we were using before.

Reply:
I believe we should set the config name to `az-analyze.config` and provide multiple search paths in this order: `/etc/az-analyze`, `$HOME/.config/az-analyze` and `.`. This would provide for global, per-user and current working directory based configurations and is pretty typical of apps for config management.